sqlitecluster/SQLiteNode.cpp

```diff
@@ -2774,3 +2823,16 @@ void SQLiteNode::_sendStandupResponse(SQLitePeer* peer, const SData& message) {
     }
     _sendToPeer(peer, response);
 }
+
+void SQLiteNode::_dieIfForkedFromCluster() {
+    size_t forkedCount = 0;
+    for (const auto& p : _peerList) {
+        if (p->forked) {
+            forkedCount++;
+        }
+    }
+
+    if (forkedCount > ((_peerList.size() + 1) / 2)) {
```

Comment:
Should we only count full peers here?  Or are we counting permafollowers here recognizing that in a bind, we can upgrade it to be a full peer, and thus we needn't necessarily recover this node?

Reply:
I am going to update this to count full peers.